SpaceCommanderDummy.m

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
```

Comment:
Looks good, a couple more things, this file needs a Square copyright header (see the testing files for example). Also, while you're add it, can you rename this file and `@implementation` to  `CocoaPodsDummy`